src/lib/components/ui/ILSMenu/index.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from "react-redux";
+import ILSMenuComponent from "./ILSMenu";
+// import { addNotification } from "@components/Notifications/state/actions";
```

Comment:
Very dangerous to have code comment out IMHO, we will forget some. That's why I was proposing to stop working on UI and to move a block of components.